src/styles/minimal/pretty.jl

```diff
@@ -11,10 +11,10 @@ function options(style::MinimalStyle)
         indent = 4,
         annotate_untyped_fields_with_any = false,
         join_lines_based_on_source = true,
-        trailing_comma = nothing,
+        trailing_comma = "nothing",
```

Comment:
these not stay as is, the issue is documenting the TOML file. In that file `nothing` needs to be surrounded with quotes but everything else remains as is.

Reply:
I'm no longer interested. I'll also unsubscribe from this PR.